app/assets/stylesheets/finder_frontend.scss

```diff
@@ -233,16 +233,91 @@
     .result-info {
       @include core-16;
       vertical-align: baseline;
-      border-bottom: solid 1px $border-colour;
       margin: 0 0 $gutter-half 0;
-      padding-bottom: $gutter-half;
 
       .result-count {
         @include bold-48;
         margin-right: $gutter-one-third / 2;
       }
     }
 
+    .filters {
+      margin-top: $gutter-one-third;
+      margin-bottom: $gutter;
+    }
+
+    .facet-tag-group {
```

Comment:
Did you consider BEM for this naming convention? I think it would work better, and would reduce the need to nest some of this CSS.